apps/costcontrol/js/app.js

```diff
@@ -46,7 +46,8 @@ var CostControlApp = (function() {
 
   var costcontrol,
       initialized = false,
-      hashFromLastRun;
+      hashFromLastRun,
+      hashFromNotification;
```

Comment:
This variable is necessary if we want to use the 'type' field  that it is sent on the notifications. 
In case of don't use this field, the method updateUI overrides the hash and load the default view
. Currently this is not a problem, but it could be on the future.